com.unity.render-pipelines.universal/Runtime/UniversalRenderPipeline.cs

```diff
@@ -503,6 +503,12 @@ static void RenderCameraStack(ScriptableRenderContext context, Camera baseCamera
                     // Helper function for updating cameraData with xrPass Data
                     m_XRSystem.UpdateCameraData(ref baseCameraData, baseCameraData.xr);
                 }
+
+                // Update volume manager to use baseCamera's settings for XR multipass rendering.
+                if (baseCameraData.xr.multipassId > 0)
```

Comment:
Let's move this block of code into the previous one that checks for xr.enabled ?

Reply:
Addressed in dd10df2